### PR TITLE
YC3GlILa - Add json-schema for passport check request payload JSON to API reference 

### PR DESCRIPTION
**Pull request by dbes-gds**
## Why

The json-schema provided can be used to validate json payloads are structured and formatted correctly.

## What

Add json-schema for passport check request payload JSON to API reference with links to validators.

### CODE REVIEW

File: source/code/dcs-passport-check-request-payload.json-schema.json [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    "requestId": {
+      "description": "String containing an RFC 4122 compliant UUID which identifies a single request within a session.",
+      "type": "string",
+      "pattern": "^[0-9a-fA-F]{8}\\-[0-9a-fA-F]{4}\\-[0-9a-fA-F]{4}\\-[0-9a-fA-F]{4}\\-[0-9a-fA-F]{12}$"
+    },
+    "timestamp": {
+      "description": "String in ISO 8601 format which identifies when the request took place.",
+      "type": "string",
+      "format": "date-time",
+      "pattern": "^[0-9]{4}-(0[1-9]|1[0-2])-(0[1-9]|[1-2][0-9]|3[0-1])T(2[0-3]|[01][0-9]):[0-5][0-9]:[0-5][0-9]\\.[0-9][0-9][0-9]Z$"
```
Comment (bjgill) [inline source/code/dcs-passport-check-request-payload.json-schema.json]: Is this correct? This is for a date-time in UTC ('Z'). DCS also supports (and some IDPs current send) timestamps with a UTC offset...

Comment (dbes-gds) [reply source/code/dcs-passport-check-request-payload.json-schema.json]: This is the format in EvidenceDTO:

    @NotBlank
    @DateFormat(format = "yyyy-MM-dd'T'HH:mm:ss.SSSZ")
    private final String timestamp;

Comment (dbes-gds) [reply source/code/dcs-passport-check-request-payload.json-schema.json]: though stub-reg appears to accept 2020-02-28T16:03:30.45+01:00

Comment (dbes-gds) [reply source/code/dcs-passport-check-request-payload.json-schema.json]: removed the regex as discussed.

**Pull request merged**